web/packages/quick-editor/src/index.ts

```diff
@@ -0,0 +1,78 @@
+export type GravatarQuickEditorOptions = {
+	avatarSelector?: string;
+	scope?: string[];
```

Comment:
We could provide a more specific type for a better DX. 

```suggestion
	scope?: ( 'avatars' | 'about' | '...' )[];
```

Reply:
✅